vector/src/main/java/im/vector/adapters/VectorParticipantsAdapter.java

```diff
@@ -386,6 +385,17 @@ private static boolean match(ParticipantAdapterItem item, String pattern) {
         return item.startsWith(pattern);
     }
 
+    /**
+     * Update the list of user ids already selected.
+     * The usersId can be either emails or matrixId,
+     * depending on whether we are in the case INVITE or NEW_ROOM.
+     *
+     * @param selectedUserIds    the list of already selected usersIds.
+     */
+    public void setSelectedUserIds(ArrayList<String> selectedUserIds) {
+        mCurrentSelectedUsers = new ArrayList<>(selectedUserIds);
```

Comment:
selectedUserIds  should be a List instead of ArrayList
mCurrentSelectedUsers should be private now